### ISSUE TITLE
Preventing Zombies and Skeletons to drop their weapons and armor on death

### ISSUE THREAD

**Issue by toffi3** [Glowing0v3rlord/Zombie-Piglins-Dont-Drop-Swords#2]
Hi, i hope you are some where around here still because i wanted to ask about the thing in the title because i was messing with that up lately and i am not sure if i don't it right. The whole "tick" function for me looks like this right now.
execute as @e[type=minecraft:zombified_piglin] run data modify entity @s HandDropChances set value [-327.67f,-327.67f]
execute as @e[type=minecraft:wither_skeleton] run data modify entity @s HandDropChances set value [-327.67f,-327.67f]
execute as @e[type=minecraft:skeleton] run data modify entity @s HandDropChances set value [-327.67f,-327.67f]
execute as @e[type=minecraft:skeleton] run data modify entity @s FeetDropChances set value [-327.67f,-327.67f]
execute as @e[type=minecraft:skeleton] run data modify entity @s LegsDropChances set value [-327.67f,-327.67f]
execute as @e[type=minecraft:skeleton] run data modify entity @s ChestDropChances set value [-327.67f,-327.67f]
execute as @e[type=minecraft:skeleton] run data modify entity @s HeadDropChances set value [-327.67f,-327.67f]
execute as @e[type=minecraft:zombie] run data modify entity @s HandDropChances set value [-327.67f,-327.67f]
execute as @e[type=minecraft:zombie] run data modify entity @s FeetDropChances set value [-327.67f,-327.67f]
execute as @e[type=minecraft:zombie] run data modify entity @s LegsDropChances set value [-327.67f,-327.67f]
execute as @e[type=minecraft:zombie] run data modify entity @s ChestDropChances set value [-327.67f,-327.67f]
execute as @e[type=minecraft:zombie] run data modify entity @s HeadDropChances set value [-327.67f,-327.67f]

Tell me if everything is correct here, thank you for making the datapack for me again

**Comment by Glowing0v3rlord** [comment]
No, that's not right. Check the wiki to see how to modify armor drop chances: https://minecraft.fandom.com/wiki/Tutorials/Command_NBT_tags#ArmorDropChances
Sorry I haven't worked on this project in a while, I don't have interest in continuing it right now. Maybe if this gets some more attention then I may change my mind.
